scripts/environment_setup.sh

```diff
@@ -15,11 +15,11 @@ sudo apt-get update && sudo apt-get install -y awscli docker-ce docker-ce-cli co
 sudo curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.8/bin/linux/amd64/kubectl
 sudo chmod +x /usr/bin/kubectl
 
-echo -n ${K8S_CLUSTER_CERT} | base64 -d > ./ca.crt
-kubectl config set-cluster ${K8S_CLUSTER_NAME} --certificate-authority=./ca.crt --server=https://api.${K8S_CLUSTER_NAME}
-kubectl config set-credentials ${SERVICE_ACCOUNT} --token=${K8S_TOKEN}
-kubectl config set-context ${K8S_CLUSTER_NAME} --cluster=${K8S_CLUSTER_NAME} --user=${SERVICE_ACCOUNT} --namespace=${K8S_NAMESPACE}
-kubectl config use-context ${K8S_CLUSTER_NAME}
+echo -n ${K8S_CLUSTER_LIVE1_CERT} | base64 -d > ./ca.crt
+kubectl config set-cluster ${K8S_CLUSTER_LIVE1_NAME} --certificate-authority=./ca.crt --server=https://api.${K8S_CLUSTER_LIVE1_NAME}
```

Comment:
This caught me out on the HMRC change -  for the new cluster the argument to `--server` doesn't need to be prefixed with `api.`, so this should just be 

`--server=https://${K8S_CLUSTER_NAME}` 

See: https://github.com/ministryofjustice/laa-hmrc-interface-service-api/pull/238/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47